tensorflow/compiler/xla/metric_table_report.cc

```diff
@@ -159,9 +161,10 @@ void MetricTableReport::AppendCategoryTable() {
 
     // Show the top entries in the category.
     const char* const kIndentPrefix = "                              * ";
-    int64 entries_to_show = std::min<int64>(max_entries_per_category_to_show_,
+    const int64 entries_to_show = std::min<int64>(max_entries_per_category_to_show_,
```

Comment:
This shouldn't have been made `const` due to line 170: being `++entries_to_show`

Reply:
updated.